client/app/src/main/java/com/microsoft/research/karya/ui/base/BaseActivity.kt

```diff
@@ -357,7 +359,7 @@ abstract class BaseActivity(
     /**
      * Request focus on a text field and show the keyboard
      */
-    protected fun requestSoftKeyFocus(eT: EditText) {
+    fun requestSoftKeyFocus(eT: EditText) {
```

Comment:
Remove this method and use `Activity.requestSoftFocus` extension everywhere instead.

Reply:
Yeah, that's in the backlog for this week

Reply:
Cool